include/grpc++/impl/codegen/call.h

```diff
@@ -592,13 +594,21 @@ class CallOpClientRecvStatus {
                            grpc::string(GRPC_SLICE_START_PTR(error_message_),
                                         GRPC_SLICE_END_PTR(error_message_)),
                            binary_error_details);
+    client_context_->set_debug_error_string(
+        debug_error_string_ != nullptr ? grpc::string(debug_error_string_)
```

Comment:
I think you do not need to explicitly construct a grpc::string here, but I assume compiler may be able to optimize it out for you.

Reply:
Done